R/glmnet-engines.R

```diff
@@ -385,23 +386,19 @@ format_glmnet_multinom_class <- function(pred, penalty, lvl, n_obs) {
   # when using `predict()`, allow for a single lambda
   if (!multi) {
     if (length(penalty) != 1) {
-      rlang::abort(
-        glue::glue(
-          "`penalty` should be a single numeric value. `multi_predict()` ",
-          "can be used to get multiple predictions per row of data."
-        )
-      )
+      cli::cli_abort(c(
+        "{.var penalty} should be a single numeric value. {.fn multi_predict}
+        can be used to get multiple predictions per row of data."
+      ))
     }
   }
 
   if (length(object$fit$lambda) == 1 && penalty != object$fit$lambda) {
-    rlang::abort(
-      glue::glue(
-        "The glmnet model was fit with a single penalty value of ",
-        "{object$fit$lambda}. Predicting with a value of {penalty} ",
-        "will give incorrect results from `glmnet()`."
-      )
-    )
+    cli::cli_abort(c(
+      "The glmnet model was fit with a single penalty value of
+      {.var object$fit$lambda}. Predicting with a value of {.envar penalty}
```

Comment:
```suggestion
      {.arg object$fit$lambda}. Predicting with a value of {.arg penalty}
```

Not the fault of this PR (so feel free to ignore), but I find this message a bit unclear